### ISSUE TITLE
Mock 4a help

### ISSUE THREAD

**Issue by morksen123** [nus-cs2030/2122-s2#608]
ik I'm supposed to do something like consumer.accept(t) but how do iterate through every number from generate? 
Does anyone mind showing their code? thanks

**Comment by GraceYongXM** [comment]
can have a for loop in forEach!

**Comment by shotnothing** [comment]
can also do forEach() in the stream

**Comment by morksen123** [comment]
> can have a for loop in forEach!

something like the 'sample' answer?

**Comment by calvinseptyanto** [comment]
Can use for loop!
for (int i = 0; i < n; i++){
consumer.accept(...);
}

**Comment by shotnothing** [comment]
> @calvinseptyanto Issue with that implementation is how to combine them into one line?

why need one line?

**Comment by shotnothing** [comment]
> > can have a for loop in forEach!
> 
> something like the 'sample' answer?

sample answer?

**Comment by morksen123** [comment]
> > > can have a for loop in forEach!
> > 
> > 
> > something like the 'sample' answer?
> 
> sample answer?

Like the answer written by the random guy in the file downloaded

**Comment by shotnothing** [comment]
> > > > can have a for loop in forEach!
> > > 
> > > 
> > > something like the 'sample' answer?
> > 
> > 
> > sample answer?
> 
> Like the answer written by the random guy in the file downloaded

err no all the answers were super whack haha

i just iterated a dummy variable thru N times, and had a .forEach() at the end to run the consumer accept get

**Comment by aldrichwilliams23** [comment]
@shotnothing Oh, my understanding from the paper is, since it's written as 2 0 6 9 5, your output need to be one line

But the code above, where you for loop and in each for loop you have consumer.accept(..) would produce

2
0
6
9
5

**Comment by morksen123** [comment]
> @shotnothing Oh, my understanding from the paper is, since it's written as 2 0 6 9 5, your output need to be one line
> 
> But the code above, where you for loop and in each for loop you have consumer.accept(..) would produce
> 
> 2 0 6 9 5

Hey, its System.out.print not System.out.printl. so a loop construct will work too

**Comment by aldrichwilliams23** [comment]
@morksen123 Woops, thanks for pointing that out!

**Comment by shotnothing** [comment]
> @shotnothing Oh, my understanding from the paper is, since it's written as 2 0 6 9 5, your output need to be one line
> 
> But the code above, where you for loop and in each for loop you have consumer.accept(..) would produce
> 
> 2 0 6 9 5

Yeah I mean even with println, you could always just reduce it to a single string and print that once right at the end
